### PR TITLE
Add data point values E2E tests

### PR DESCRIPTION
**Pull request by WiNloSt**
Epic: https://github.com/metabase/metabase/issues/24759

### CODE REVIEW

File: frontend/src/metabase/static-viz/components/XYChart/Values/Values.tsx [hunk 1]
```diff
@@ -150,10 +150,10 @@ export default function Values({
               {shouldRenderDataPoint && (
                 <circle
                   key={index}
-                  r={2}
+                  r={3}
```
Comment (WiNloSt) [inline frontend/src/metabase/static-viz/components/XYChart/Values/Values.tsx:153]: Make data point circles easier to see. And looks similar size as the app viz data point circles.

#### After
image

#### Before
image

File: frontend/test/metabase-visual/static-visualizations/line-area-bar-combo.cy.spec.js [hunk 2]
```diff
@@ -22,14 +22,14 @@ describe("static visualizations", () => {
     setupSMTP();
   });
 
-  visualizationTypes.map(type => {
-    it(`${type} chart`, () => {
-      const dashboardName = `${type} charts dashboard`;
+  visualizationTypes.map(visualizationType => {
```
Comment (WiNloSt) [inline frontend/test/metabase-visual/static-visualizations/line-area-bar-combo.cy.spec.js:25]: rename variable for clarity

File: frontend/test/metabase-visual/static-visualizations/line-area-bar-combo.cy.spec.js [hunk 3]
```diff
@@ -55,15 +55,16 @@ function createOneDimensionTwoMetricsQuestion(display) {
...
       "graph.metrics": ["count", "avg"],
+      "graph.show_values": true,
     },
     display: display,
     database: SAMPLE_DB_ID,
   };
 }
 
-function createOneMetricTwoDimensionsQuestion(display) {
+function createOneMetricTwoDimensionsQuestion(visualizationType) {
```
Comment (WiNloSt) [inline frontend/test/metabase-visual/static-visualizations/line-area-bar-combo.cy.spec.js:65]: rename argument for consistency with its usages

**Review by alxnddr — APPROVED** [review]
It is not related to this PR, it is good to merge, but some charts from the specs still look overcrowded with ticks:
Screen Shot 2022-09-09 at 7 57 40 PM

Seems like we need to dramatically decrease the number of labels

**Comment by WiNloSt** [comment]
> **[alxnddr](/alxnddr) ** left a comment

Thanks for the review. I kind of know why the algorithm for overlapping values only works for the same x coordinate. It seems in the screenshot, each bar is considered a different x coordinate, so it's free to render side by side like that. I'll try to see how the app viz handles a case like this (I already try to make the static viz behave as close to the app viz as possible).

**Pull request merged**